lib/railway_ipc/tasks/generate_migrations.rake

```diff
@@ -1,8 +1,24 @@
+require 'fileutils'
+
 namespace :railway_ipc do
   namespace :generate do
+    desc "Generates migrations to store Railway messages"
     task :migrations do
       if defined?(ActiveRecord::Base)
-        puts "working"
+        puts "generating Railway IPC table migrations"
+
+        seconds = 0
+        gem_path = Gem.loaded_specs['railway-ipc'].full_gem_path
+        folder_dest = "#{Rails.root.to_s}/db/migrate"
+        FileUtils.mkdir_p(folder_dest)
```

Comment:
`mkdir -p` since when a new rails app is created, there's no `migrate` dir in `db`.